### DISCUSSION TITLE
DS3 pressure-sensitive buttons @ PS2

### DISCUSSION THREAD

**Discussion by zenq01** [darthcloud/BlueRetro#260] [General]
Hello,
is there an option to enable recognition of pressure sensitive buttons in DS3 connected to PS2 via blueretro? now all buttons only work in zero-one (except for the analog sticks)

**Comment by darthcloud** [comment]
No, not yet.

**Reply by andrebrait** [reply]
Is that still the case? Just checking whether it was addressed but the issue was still open for some reason.

**Reply by zenq01** [reply]
still waiting for it ;)

**Reply by rodrigo-guim** [reply]
@darthcloud do you have any directions for this? So I can take a look and try to open a PR

**Reply by sausehuhn** [reply]
@darthcloud Also wondering if this is on the roadmap for DS3/DS4 :) Basically planning to play a very small selection of games. One of them uses pressure sensivity for a game mechanic that can make or break your races.

**Reply by th3drk0ne** [reply]
Going to replace all my blueretro adapters now with the brooks wingman ps2 as they have this feature enabled when using a DualShock 3, which is a pitty as I love this ecosystem and I would much prefer not to have to use the brooks but for  100% game coverage I have no choice

**Reply by guimaraesrodrigues** [reply]
@th3drk0ne what benefits does your comment bring to the discussion of this topic?

**Comment by th3drk0ne** [comment]
What discussion? I would love this feature request to be up ticked and
actually worked on. I am a firm believer in this project I was just
disappointed that this has has no prioritisation. Instead of being
aggressive towards me please retarget your efforts into adding this feature

On Mon 19 Jun 2023 at 14:23, Rodrigo Guimarães Rodrigues de Almeida <
***@***.***> wrote:

> @th3drk0ne <https://github.com/th3drk0ne> what benefits does your comment
> bring to the discussion of this topic?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/darthcloud/BlueRetro/discussions/260#discussioncomment-6219430>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABLJ4VJUV7PJXORYPGZI3ILXMBHEDANCNFSM5TZ62D6A>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Karl

**Reply by guimaraesrodrigues** [reply]
ok, Karen

**Reply by th3drk0ne** [reply]
Only a Karen would say that

**Comment by darthcloud** [comment]
This beta add support for ps3 ctrl pressure button while using a PS1/PS2 system:
https://github.com/darthcloud/BlueRetro/actions/runs/7398955532

Everything can be remap at will as usual and the remap will be analog if the source and destination are analog.
Turbo work as well on pressure buttons.

If you use a modern controller without pressure button, you can map a joystick direction or one analog trigger to one of the PS2 pressure button and it will be analog.

**Reply by afritsch89** [reply]
This is amazing, can't wait to try it out!!

**Reply by Razmann4k** [reply]
~Has this feature been added to the main branch? This link is now dead.~ Nevermind, I see pressure sensitive button support on PS3 controllers was added in the 2024-02-18 release.
